warehouse/admin/templates/admin/projects/detail.html

```diff
@@ -108,4 +108,28 @@ <h3 class="project-name text-center">{{ project.name }}</h3>
         </div>
       </div>
   </div>
+  <div class="box box-secondary">
+    <div class="box-header with-border">
+      <h3 class="box-title">Blacklist Project</h3>
+    </div>
+
+    <form method="GET" action="{{ request.route_path('admin.blacklist.add') }}">
+      <div class="box-body">
+        <div class="form-group">
+          <input name="project" class="form-control" id="blacklistedProject" type="hidden" readonly="readonly" value="{{ project.normalized_name }}" placeholder="{{ project.name }}">
```

Comment:
The ``class``, ``id``, ``readonly``, and ``placeholder`` attributes are no longer needed now that this is a hidden field.